### DISCUSSION TITLE
ERROR:root:Failed with output:error: pathspec 'origin/v10.0.0' did not match any file(s) known to git.

### DISCUSSION THREAD

**Discussion by benjamin-cash** [dtcenter/METplus#1056] [Installation] [answered]
I am installing METplus on Expanse at SDSC and saw the following error message go by early on in the process:

ERROR:root:Failed with output:
    error: pathspec 'origin/v10.0.0' did not match any file(s) known to git.

Is this something I need to be concerned about?

**Comment by benjamin-cash** [comment]
@georgemccabe - Is this issue also due to the problem where the build scripts need to be updated?

**Comment by georgemccabe** [comment, marked as answer]
> 
> 
> @georgemccabe - Is this issue also due to the problem where the build scripts need to be updated?

Hi @benjamin-cash, yes, this is related to the issue with the build components script. The GitHub issue to fix this script is here: #1068.

The steps to install a MET release are outlined in the other discussion topic here: https://github.com/dtcenter/METplus/discussions/1059#discussioncomment-1140418
We will update the METplus User's Guide instructions with these steps soon.

This specific error comes from the manage_externals logic to obtain the MET source code. The tag v10.0.0 is used but manage_externals must be provided a branch and not a tag. The correct branch to use is main_v10.0, which will contain all bug fixes applied to 10.0.0 even if a bug fix release, i.e. 10.0.1, has been created. However, the recommended approach to installing a MET release is to use the modified release tar file that is found on the METplus Components Downloads page. See the comments in #1059 for more details.

**Reply by benjamin-cash** [reply]
Hi @georgemccabe  - did you get a chance to look at my follow-up in #1059?

**Reply by georgemccabe** [reply]
@benjamin-cash, yes, I am replying to that topic now. I apologize for the delay in response. I have been out of the office for the last few work days.

**Reply by benjamin-cash** [reply]
No worries, and thanks again. :)
